### PR TITLE
templates: bump kommander and adds a draft template for konvoy-ui

### PR DESCRIPTION
**Pull request by hectorj2f**
- [x] Bumps kommander versions to use the latest changes in mesosphere/charts (to be merged)
- [x] Adds a dummy `konvoy-ui` addon to the template `(THIS TEMPLATE NEEDS TO BE FILLED WITH THE RIGHT VALUES)`

### CODE REVIEW

**Comment by hectorj2f** [comment]
cc @alejandroEsc @s12chung

**Comment by jaxesn** [comment]
Here is the ticket with is needed, not sure all of this will implemented in this repo or not just throwing it out there...

https://jira.mesosphere.com/browse/DCOS-55832

**Comment by alejandroEsc** [comment]
I would like to hold off merging this for a bit and have a release of kubeaddons configs before this is added.

**Comment by alejandroEsc** [comment]
@natmegs can you have a look?

File: templates/konvoy-ui.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoy-ui
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: konvoy-ui
+  annotations:
+    appversion.kubeaddons.mesosphere.io/kommander: "1.40.0"
+    endpoint.kubeaddons.mesosphere.io/landing: /landing
```
Comment (natmegs) [inline templates/konvoy-ui.yaml]: This looks different than the endpoint annotations in other templates - is the `/landing` postfix unique to konvoy-ui? Others seem to use the `kubeaddons.mesosphere.io/name` label value

File: templates/konvoy-ui.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      ---
+      service:
+        name: konvoy-ui
+      ingress:
+        paths:
+          # konvoy-ui
+          - backend:
+              serviceName: konvoy-ui
+              servicePort: 4000
+            path: /landing
```
Comment (jieyu) [inline templates/konvoy-ui.yaml]: what about the ops portal page? i.e., `/ops/portal/`

File: templates/konvoy-ui.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: kommander
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.1.6
+    values: |
```
Comment (DanielMSchmidt) [inline templates/konvoy-ui.yaml]: I just noticed we need the environment variable `MODE=konvoy` in the kommander chart. I would change the chart to allow the user to pass down environment variables through the values.

**Comment by hectorj2f** [comment]
Closing this PR in favor of #128

**Pull request closed without merging**